dom/src/mockDOMSource.ts

```diff
@@ -1,81 +1,81 @@
 import {StreamAdapter, DevToolEnabledSource} from '@cycle/base';
+import {DOMSourceable, GenericStream, EventsFnOptions} from './DOMSourceable';
 import {VNode} from './interfaces';
-import xsSA from '@cycle/xstream-adapter';
-import {DOMSource, EventsFnOptions} from './DOMSource';
+import xsAdapter from '@cycle/xstream-adapter';
 import xs from 'xstream';
 
-export type GenericStream = any;
-export type ElementStream = any;
-export type EventStream = any;
-
 export type MockConfig = {
   [name: string]: GenericStream | MockConfig;
   elements?: GenericStream;
 }
 
-const SCOPE_PREFIX = '___';
+const CYCLE_SOURCE = `MockedDOM`;
```

Comment:
Good for making this const so there is no duplication of the raw string, I think we can just rename to be "SOURCE_NAME" since the purpose of this is for DevTools. "CYCLE_SOURCE" is a bit too generic, and at this point we don't need "CYCLE" in front because everything in this codebase is related to Cycle.js